spec/source/locale-sensitive-functions.html

```diff
@@ -0,0 +1,326 @@
+    </p>
+  </emu-note>
+
+  <emu-clause id="sec-properties-of-the-string-prototype-object">
+    <h1>Properties of the String Prototype Object</h1>
+
+    <emu-clause id="sec-String.prototype.localeCompare">
+      <h1>String.prototype.localeCompare (that [, locales [, options ]])</h1>
+
+      <p>
+        This definition supersedes the definition provided in ES2015, 21.1.3.10.
+      </p>
+
+      <p>
+        When the *localeCompare* method is called with argument _that_ and optional arguments _locales_, and _options_, the following steps are taken:
+      </p>
+
+      <emu-alg aoid="alg-String.prototype.localeCompare">
+        1. Let _O_ be RequireObjectCoercible(*this* value).
+        2. Let _S_ be ToString(_O_).
+        3. ReturnIfAbrupt(_S_).
+        4. Let _That_ be ToString(_that_).
+        5. ReturnIfAbrupt(_That_).
+        6. Let _collator_ be Construct(*%Collator%*, «*locales*, *options*»).
+        7. ReturnIfAbrupt(_collator_).
+        8. Return CompareStrings(_collator_, _S_, _That_)
+      </emu-alg>
+
+      <p>
+        The value of the *length* property of the *localeCompare* method is 1.
+      </p>
+
+      <emu-note>
+        <p>
+          The *localeCompare* method itself is not directly suitable as an argument to *Array.prototype.sort* because the latter requires a function of two arguments.
+        </p>
+      </emu-note>
+
+      <emu-note>
+        <p>
+          The *localeCompare* function is intentionally generic; it does not require that its *this* value be a String object. Therefore, it can be transferred to other kinds of objects for use as a method.
+        </p>
+      </emu-note>
+
+    </emu-clause>
+
+    <emu-clause id="sec-String.prototype.toLocaleLowerCase">
+      <h1>String.prototype.toLocaleLowerCase ([locales])</h1>
+
+      <p>
+        This definition supersedes the definition provided in ES2015, 21.1.3.20.
+      </p>
+
+      <p>
+        This function interprets a string value as a sequence of code points, as described in ES2015, 6.1.4. The following steps are taken:
+      </p>
+
+      <emu-alg aoid="alg-String.prototype.toLocaleLowerCase">
+        1. Let _O_ be RequireObjectCoercible(*this* value).
+        2. Let _S_ be ToString(_O_).
+        3. ReturnIfAbrupt(_S_).
+        4. Let _requestedLocales_ be CanonicalizeLocaleList(_locales_).
+        5. ReturnIfAbrupt(_requestedLocales_).
+        6. Let _len_ be the number of elements in _requestedLocales_.
+        7. If _len_ > 0, then
+          1. Let _requestedLocale_ be the first element of _requestedLocales_.
+        8. Else
+          1. Let _requestedLocale_ be DefaultLocale().
+        9. Let _noExtensionsLocale_ be the String value that is _requestedLocale_ with all Unicode locale extension sequences (6.2.1) removed.
```

Comment:
cross link to 6.2.1 requires section 6 to be merged, otherwise ecmarkup fails to build.